### ISSUE TITLE
Decide how to allocate tokens

### ISSUE THREAD

**Issue by holkeb** [Network-Goods/hypercerts#106]
We need to decide what fraction of the hypercerts will be distributed to the funders and how much the team keeps.

**Comment by holkeb** [comment]
Let's suppose we have two project, which did a similar amount of work in the time period of work (let's just use hours as a measurement for the sake of the example). In the round they raise very different amounts (retail + matching amount), e.g.

Project 1
- raised: $10,000
- amount of work: 100h

Project 2: 
- raised: $50 
- amount of work: 100h

Both projects mint a hypercert with 10,000 tokens. Now we can define some formula to adjust that they raised different amounts, e.g. project 2 could retain some more tokens. In this case that makes sense as they put in the same amount of work. However, we actually don't know the "amount of work". Hence, there could be a third project:

Project 3:
- raised: $50
- amount of work: 1h

So, adjusting for the raised amount doesn't actually make sense if we don't know the amount of work.

The solution that I see is that we allow projects to change the amount how much they keep – acknowledging that this is difficult, but I think it is more fair to let projects decide than to implement something that is definitely off. There also won't be a right answer on the amount.

**Comment by holkeb** [comment]
Additionally, suppose...

Project 4:
- raised previously from some other funder: $10,000
- raised during the grants round: $50

Now we might assign that 50% of the hypercert is distributed to all funders from the round, i.e. 50% is given out for $50. Then they want to pass on some of the hypercert to their previous funder, but there isn't any way that the project can do that in a reasonable proportion.

**Comment by MF416** [comment]
@holkeb is there a hack where we just say something along the line of:
- The scope of work relates only to the FUNDING of the project through Gitcoin
- That way, the % fractions are proportional to this funding round only
- Contributing teams are acknowledged but not included in %

Example:

- Project on Github raises $10k total
- $5k matching from Gitcoin -> Gitcoin gets 50% fractional hypercert
- $5k from 10 Community members ($500 each) -> 5% fractional hypercert each
- Team is acknowledged, but scope is defined as "Funding work only"

Cleanest from an ops sense ... deciding % value might present too much of a barrier for a lot of teams. But also a good data point on a necessary step in full HC implementation?

**Comment by holkeb** [comment]
I don't know conceptually what "The scope of work relates only to the FUNDING of the project through Gitcoin" means.

In general keeping the _token amount_ and _funds raised_ constant and adjusting the `scope of work` is also a solution, but potentially a more difficult one to explain.

I could see different implementations like: The default is that 50% of the tokens are distributed to funders and projects are able to either adjust this to 25% (keeping more themselves) or to 75%. That should be mentally quite simple. We can give them some explanation.

> By default we suggest that you distribute 50% of the hypercert fractions to the funders from the Gitcoin Grants Round. You will keep the other 50%, which you can distribute to your team, to other funders who have funded this work previously or to funders who will fund this work retrospectively in the future. You can choose to adjust the percentage to 25% or 75%. 
> - We suggest to only decrease it to 25% if the funds raised during the Gitcoin Grants Round represent a small part of your overall budget for the past work described by the hypercert.
> - We suggest to increase it to 75% if the funds raised during the Gitcoin Grants Round represent a large part of your overall budget for the past work described by the hypercert.

**Comment by MF416** [comment]
Sorry catching up @holkeb 

Either by indicating in the `scope of work` field or `scope of impact` field that the hypercerts here are restricted to the work of funding the project or the impact created by funding only.  Agreed, it's clumsier / hard to explain for users at the benefit of being easier to implement.

Your solution seems like a good balance --> what I might suggest is we run this past a friendly potential project ahead of time to get their reaction?

**Comment by holkeb** [comment]
Agreed to get it in front of projects next week

**Comment by ccerv1** [comment]
Going to move this issue to "blocked" until we have some user interviews. 

Sounds we have the following options:

1. All projects should allocate 100% of their hypercert fractions to funders
2. All projects should allocate the same share of fractions to funders and retain the balance for themselves (eg 75/25) 
3. Each project is given a unique allocation based on data from the grants round (eg, the top project allocates all its fractions to funders, whereas a project with very little funding keeps most of its fractions)
4. All projects are free to choose whatever allocation they want

We will get some user feedback and then weigh pros/cons.

**Comment by ccerv1** [comment]
@holkeb keeping this issue visible. We are currently weighing options 1 and 3 it seems like.

**Comment by MF416** [comment]
@ccerv1 have we tested the options with any potential projects yet?

1) seems easier to implement, 3) would be harder but probably more informative ...

**Comment by ccerv1** [comment]
@MF416 I have held off speaking to projects until we have legal sign-off. Hopefully this gets unblocked Monday!

**Comment by ccerv1** [comment]
Closing this out, as we're going with 1.
In the event that we get feedback that causes us to change this decision, we can open a new issue.
